src/components/TokenInput/TokenInput.tsx

```diff
@@ -8,7 +8,7 @@ export interface TokenInputProps {
   isDisabled?: boolean;
   isHalfAndMaxButtonsVisible?: boolean;
   title: string;
-  tokenAmount: number;
+  tokenAmount: number | string;
```

Comment:
It doesn't look right, we should ether use number or string.  Mixing makes consumption complicated.